projects/wx.cmake

```diff
@@ -111,6 +111,14 @@ function(build_wxv)
     if(${CMAKE_C_COMPILER_ID} MATCHES "Clang")
       list(APPEND XP_CONFIGURE_BASE CC=clang)
     endif()
+    execute_process(COMMAND uname --machine
+      OUTPUT_VARIABLE unameMachine
+      OUTPUT_STRIP_TRAILING_WHITESPACE
+      ERROR_VARIABLE unameErr
+      )
+    if(DEFINED unameMachine AND NOT unameErr AND unameMachine MATCHES "^aarch")
+      list(APPEND XP_CONFIGURE_BASE --build=aarch64-unknown-linux-gnu --enable-arm-neon)
```

Comment:
for aarch64 I have to specify the build type and enable the arm neon optimizations. (only effects arm 7 and greater)

Reply:
my plan is to take your changes to this file, and the `patches/wx31.patch` changes and nothing else, for now... unless you convince me that we need the `patches/wxcmake31.patch` changes

but the changes to `projects/README.md`, and `projects/wx31.cmake` and `projects/wxcmake31.cmake` aren't necessary and I'd rather keep the history and blame where it is then merge these changes and then immediately modify these files (and introduce unnecessary history and lose blame on these lines)